### ISSUE TITLE
Autoincrement models/note.js

### ISSUE THREAD

**Issue by jnschilling** [gitdagray/mern_stack_course#9]
End of the controller lesson 4, I noticed that a POST to create a new note to a valid user:
It is leading to an unending request into postman.
I commented the autoincrement plugin, the issue is not there any more :
// noteSchema.plugin(AutoIncrement, {
//     inc_field: 'ticket',
//     id: 'ticketNums',
//     start_seq: 500
// })
is this plugin still supported ?

**Comment by eshaparenko** [comment]
Hi. Also faced this issue.
There is opened bug https://github.com/ramiel/mongoose-sequence/issues/133

Checked with previous version of monggose ("mongoose": "^6.11.0",). Issue is not reproduced.

**Comment by RobertBeit** [comment]
yes like eshaparenko said the best workaround is to just install the previous version of mongoose using npm install mongoose@6.11.0

**Comment by Pubudu-Perera** [comment]
had the same issue.  Problem is the mongoose version.  stick into version 6.5.0
